perform__hashing.py

```
@@ -0,0 +1,29 @@
+import hashlib
+import secrets
+
+# 1. Define the password (string)
+password = "06032022ArteAnas!"
```

Ruff (0.14.5)

Possible hardcoded password assigned to: "password"

(S105)